cluster/gke/util.sh

```diff
@@ -241,9 +241,15 @@ function detect-minion-names {
   echo "... in gke:detect-minion-names()" >&2
   detect-project
   detect-node-instance-group
-  MINION_NAMES=($(gcloud compute instance-groups managed list-instances \
-    "${NODE_INSTANCE_GROUP}" --zone "${ZONE}" --project "${PROJECT}" \
-    | cut -d'/' -f11))
+  if [[ ! -z "${DOGFOOD_GLOUD:-}" ]]; then
```

Comment:
Is using this env variable the right choice? Will this eventually affect the stable release of gcloud?